keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/titus/verification/TestContainerVerificationEvaluator.kt

```diff
@@ -36,7 +36,8 @@ class TestContainerVerificationEvaluator(
     verification: Verification,
     metadata: Map<String, Any?>
   ): ConstraintStatus {
-    val taskId = metadata[TASK_ID]
+    @Suppress("UNCHECKED_CAST")
+    val taskId = (metadata[TASKS] as Iterable<String>?)?.last()
```

Comment:
Why are we picking the last?

Reply:
That's what the `json_merge` function does. If there's already a `tasks` array in the JSON it will merge the 2 together. So the last in the array will always be the most recent task. See the test in `VerificationRepositoryTests` that checks exactly this.

Reply:
Ah, right! About the test, I saw the it was checking for the size of the list, but not the order. Not a big deal, though. I understand the behavior now.